src/resources/org/apache/cassandra/schema/schema_validation_tests/test_1-7ab21491848211ee825e31532694b0d3/bb-2-bti-Digest.crc32

```diff
@@ -0,0 +1 @@
+3063591096
```

Comment:
As discussed offline, an improvement to this would be to programmatically create the schema and the sstables, so that the test also becomes more easily extensible: if we need to check a new schema we don't need to create the sstable externally and bring it into the repo. 

We could do this in a separate ticket if we are short on time.

Reply:
Agree that it would make the tests more flexible and easy to evolve. We'd need to find a reliable way to create SSTables programmatically, which in a unit test could be tricky. I would create a separate ticket for this and follow up on it when we have a chance.

Reply:
An example would be `AbstractPendingAntiCompactionTest#makeSSTables`. Should not be too difficult but I agree with creating a separate ticket for this.